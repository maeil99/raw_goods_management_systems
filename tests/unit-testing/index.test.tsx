/* eslint-disable no-undef */
/* eslint-disable import/no-unresolved */
import { render, screen } from '@testing-library/react';
import Home from '../../pages';

describe('Home', () => {
  it('renders a heading', () => {
    render(<Home />);
    const { container } = render(<Home />);
    const header = screen.findByText(/Hello World/i);

    expect(header).toBeDefined();
    expect(container).toMatchSnapshot();

    screen.debug();
  });
});
