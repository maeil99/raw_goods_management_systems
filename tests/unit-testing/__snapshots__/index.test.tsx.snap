// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders a heading 1`] = `
<div>
  <div>
    <h1>
      Hello World
    </h1>
  </div>
</div>
`;
